libnmstate/state.py

```diff
@@ -264,3 +269,41 @@ def dict_update(origin_data, to_merge_data):
         else:
             origin_data[key] = val
     return origin_data
+
+
+class LinuxBridgeState(object):
+    DEFAULT_STP_ENABLED = True
+    DEFAULT_PORT_STP_HAIRPIN_MODE = False
+    DEFAULT_PORT_STP_PATH_COST = 100
+    DEFAULT_PORT_STP_PRIORITY = 32
+
+    DEFAULT_PORT_STP_OPTIONS = {
```

Comment:
I wonder if having a dictionary as a default is a good idea. If objects want to use the defaults, they should probably do a deepcopy to ensure that they do not change each others port options, do they not? Therefore a method that generates defaults might be better.